### PR TITLE
SOM ranks

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Adds ranks for SOM, and makes playtime visible (it was already recorded)

I meant to do this a long time ago but never got around to it.
## Why It's Good For The Game
Playable faction having ranks is good.
## Changelog
:cl:
add: Added ranks for SOM
/:cl:

### CODE REVIEW

**Comment by Al-1ce** [comment]
SOM ranks being 1-1 with marines is kinda meh you'd think they'd have different names but w/e

**Comment by Lumipharon** [comment]
> SOM ranks being 1-1 with marines is kinda meh you'd think they'd have different names but w/e

They're not though?

File: code/datums/jobs/job/sonsofmars.dm [hunk 1]
```diff
@@ -71,6 +71,28 @@ Make your way to the cafeteria for some post-cryosleep chow, and then get equipp
...
+		if(601 to 6000) // 10hrs
+			new_human.wear_id.paygrade = "SOM_E2"
+		if(6001 to 18000) // 100 hrs
+			new_human.wear_id.paygrade = "SOM_E3"
+		if(18001 to 30000) // 300 hrs
+			new_human.wear_id.paygrade = "SOM_E4"
+		if(30001 to 60000) // 500 hrs
+			new_human.wear_id.paygrade = "SOM_E5"
+		if(60001 to INFINITY) // 1000 hrs
+			new_human.wear_id.paygrade = "SOM_S1"
```
Comment (Pariah919) [inline code/datums/jobs/job/sonsofmars.dm:94]: Should be S5, honestly. The entire joke behind 1000 hours is you get a snowflake rank nobody else gets.

**Comment by novaepee** [comment]
image

oh my god, my military brain is cringing right now

**Pull request merged**